.gitmodules

```diff
@@ -5,3 +5,6 @@
 [submodule "py2hwsw/lib/hardware/iob_system/submodules/VEXRISCV"]
 	path = py2hwsw/lib/hardware/iob_system/submodules/VEXRISCV
 	url = git@github.com:IObundle/iob-vexriscv.git
+[submodule "py2hwsw/lib/hardware/iob_system/submodules/iob-ibex"]
+	path = py2hwsw/lib/hardware/iob_system/submodules/iob-ibex
+	url = https://github.com/IObundle/iob-ibex.git
```

Comment:
The ibex submodule is using an URL with the `https` protocol, while the vexriscv submodule is using the `ssh` protocol.
We should have both submodules using the same protocol.

I'm not sure which protocol is better, but I think the `https` one may be preferable as it allows users to clone this repo without being authenticated with github.

The following comment suggests publishing submodules with the `https` protocol, but using `ssh` for developers (via git configuration): https://stackoverflow.com/a/50299434/11442904

Reply:
the comment is a good one

Reply:
The comment is good